packages/components/src/radio-control/index.tsx

```diff
@@ -73,7 +83,14 @@ export function RadioControl(
 			help={ help }
 			className={ clsx( className, 'components-radio-control' ) }
 		>
-			<VStack spacing={ 2 }>
+			<VStack
+				spacing={ 3 }
+				className={ clsx(
+					'components-radio-control__group-wrapper',
+					!! help &&
+						'components-radio-control__group-wrapper--with-help'
+				) }
```

Comment:
```suggestion
				className={ clsx( 'components-radio-control__group-wrapper', {
					'has-help': !! help,
				} ) }
```

For better adherence to our [modifier conventions](https://developer.wordpress.org/block-editor/contributors/code/coding-guidelines/#naming).

Reply:
I had forgotten about it — thank you for pointing out. Done in https://github.com/WordPress/gutenberg/pull/63751/commits/46fb3e14b7c2fe6ee08e941b0146813f94314608